test/routes/origins.js

```diff
@@ -1,10 +1,62 @@
+// @ts-nocheck
 // Copyright (c) The Linux Foundation and others. Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 
 const { expect } = require('chai')
 const httpMocks = require('node-mocks-http')
 const sinon = require('sinon')
 const originCondaRoutes = require('../../routes/originConda')
+const originMavenRoutes = require('../../routes/originMaven')
+const fs = require('fs')
+
+describe('Maven Origin routes', () => {
+  let router
+  const fixturePath = 'test/fixtures/origins/maven'
+
+  before(() => {
+    router = originMavenRoutes(true)
+  })
+
+  it('should return suggestions when incomplete group id is provided as input', async () => {
+    const partialGroupId = 'org.apache.httpcom'
+    expect(getResponse(partialGroupId)).to.be.deep.equal([
+      'httpcore',
+      'httpconn',
+      'httpcodec',
+      'httpcommons',
+      'httprox'
+    ])
+  })
+
+  it('should return list of artefacts when complete group id is provided as input', async () => {
+    const completeGroupId = 'org.apache.httpcomponents'
+    expect(getResponse(completeGroupId)).to.be.deep.equal(
+      loadFixture(`${fixturePath}/${completeGroupId}-response.json`)
+    )
+  })
+
+  it('should return blank response when group id is invalid and suggestions are not present', async () => {
+    const invalidGroupId = '12345'
+    expect(getResponse(invalidGroupId)).to.be.deep.equal([])
+  })
+
+  it('should return blank response when group id and artefact id are invalid and suggestions are not present', async () => {
+    const invalidGroupId = '12345'
+    const invalidArtifactId = '1234'
+    expect(
+      router._getSuggestions(loadFixture(`${fixturePath}/${invalidGroupId}-${invalidArtifactId}.json`), invalidGroupId)
```

Comment:
This line looks a bit too long.  I would assign a variable for loadFixture(..)

Reply:
Thank you for the feedback. I have pushed the changes.